common/contexts/src/dal/dal_context.rs

```diff
@@ -64,6 +65,10 @@ impl Layer for DalContext {
 
 #[async_trait]
 impl Accessor for DalContext {
+    async fn create(&self, args: &OpCreate) -> Result<()> {
+        self.inner.as_ref().unwrap().create(args).await
+    }
```

Comment:
The code style is wired, using make lint to do the format?

Reply:
`make lint` passed without error.

But the `unwrap` here does need some improvement, I will fix it in a new PR.

Reply:
Tracked in https://github.com/datafuselabs/databend/issues/4741